src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/config/FeatureFlagsService.kt

```diff
@@ -12,9 +12,15 @@ enum class ReviewAddedSyncMechanism {
 class FeatureFlagsService(
   @Value("\${feature.review-added-sync-mechanism}")
   val reviewAddedSyncMechanism: String,
+  @Value("\${feature.incentives-data-source-of-truth:false}")
+  val incentivesDataSourceOfTruth: Boolean
```

Comment:
You could name this something like `incentivesDbSourceOfTruth` to be a bit more explicit?

Reply:
Do you have any ideas have you to change or could do so in the next PR?

Reply:
updated